src/main/java/rx/internal/operators/OperatorIgnoreElements.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2014 Netflix, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package rx.internal.operators;
+
+import rx.Observable.Operator;
+import rx.Subscriber;
+
+public class OperatorIgnoreElements<T> implements Operator<T, T> {
+
+    private static class Holder {
+        static final OperatorIgnoreElements<?> INSTANCE = new OperatorIgnoreElements<Object>();
+    }
+    
+    @SuppressWarnings("unchecked")
+    public static <T> OperatorIgnoreElements<T> instance() {
+        return (OperatorIgnoreElements<T>) Holder.INSTANCE;
+    }
+
+    private OperatorIgnoreElements() {
+
+    }
+
+    @Override
+    public Subscriber<? super T> call(final Subscriber<? super T> child) {
+        return new Subscriber<T>(child) {
```

Comment:
This will forward the child's requests and will hang because the child doesn't receive an element and nobody is requesting more. I'd instead write this:
```java
Subscriber<T> result = new Subscribeer<T>() { ... }
child.add(result);
return result;
```